api/app/signals/apps/dsl/example_dsl.py

```diff
@@ -0,0 +1,47 @@
+import time
```

Comment:
Maybe we should have `/docs` directory inside the `dsl` app where we can move this example along with the design documentation. Furthermore this script needs access to the `DJANGO_SETTINGS_MODULE` to run, please fix.

Reply:
deleted